src/main/java/com/rcv/Tabulator.java

```diff
@@ -843,6 +871,29 @@ private void incrementTally(
     tally.put(selectedCandidateID, newTally);
   }
 
+  // function: incrementTallies
+  // purpose: transfer vote to round tally and (if valid) the precinct round tally
+  // param: tally is a round tally that we're in the process of computing
+  // param: cvr is a single cast vote record
+  // param: selectedCandidateID is the candidate this CVR's vote is going to in this round
+  // param: roundTallyByPrecinct map of precinct IDs to roundTallies
+  // param: precinct ID of precinct for current CVR
+  private void incrementTallies(
+      Map<String, BigDecimal> tally,
+      BigDecimal fractionalTransferValue,
+      String selectedCandidateID,
```

Comment:
@HEdingfield and I would both prefer to refer to candidates just as candidates (as in selectedCandidate) rather than candidateIDs in the code, since ID kind of implies we're passing around numbers.